pkg/apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -180,6 +180,10 @@ type Database struct {
 	BackupRemoteDeletePolicy string `json:"backupRemoteDeletePolicy,omitempty"`
 	// Defines the secret to be used for uploading/restoring backup.
 	// +optional
+	BackupSecretName string `json:"backupSecretName,omitempty"`
+	// Optionally enter the name of already existing Secret
+	// +optional
+	Secret                  string `json:"secret,omitempty"`
```

Comment:
I think this name is not a good one, maybe we can change that to something like `ExistingDBSecret`

Reply:
since this is already in Database struct I renamed it to `ExistingSecret`